db/fixtures/miq_product_features.yml

```diff
@@ -5123,3 +5123,36 @@
     :description: Delete Arbitration Settings
     :feature_type: admin
     :identifier: delete_arbitration_setting
+
+# Blueprints
+- :name: Blueprints
+  :description: All Blueprints Access Rules
+  :feature_type:
```

Comment:
@imtayadeway to keep feature ids consistent with others in https://github.com/ManageIQ/manageiq/blob/master/db/fixtures/miq_product_features.yml, can we change feature id prefix to be singular, so those can be changed to blueprint_xx
Also can you change following ids to match the ones in the yml as well:
blueprints_list -> blueprint_show_list
blueprints_modify -> blueprint_edit
blueprints_add -> blueprint_new